Serializer/Normalizer/ExceptionNormalizer.php

```diff
@@ -23,13 +22,13 @@
  * @author Johannes M. Schmitt <schmittjoh@gmail.com>
  * @author Florian Voutzinos <florian@voutzinos.com>
  */
-class ExceptionWrapperNormalizer implements NormalizerInterface
+class ExceptionNormalizer implements NormalizerInterface
 {
-    private $translator;
+    private $exceptionMessages;
 
-    public function __construct(TranslatorInterface $translator)
+    public function __construct($exceptionMessages)
     {
-        $this->translator = $translator;
```

Comment:
Yeah it's just a first approach ^_^ I plan to move the form normalization to another normalizer.